pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -789,3 +791,17 @@ func parseSMARTData() {
 	}
 	smartData.RawData = string(data)
 }
+
+func getCapabilities(ctx *zedagentContext) *info.Capabilities {
+	m, err := ctx.subCapabilities.Get("global")
+	if err != nil {
+		log.Warnf("ctx.subCapabilities.Get failed: %s", err)
+	} else {
```

Comment:
Nit; return nil here and no else makes it easier to read.

Reply:
Done